SR2022/2022-04-02-competition-details-reminder.md

```diff
@@ -0,0 +1,40 @@
+---
+to: Student Robotics 2022
+subject: Competition details and important reminders
+---
+
+Hello all!
+
+It's just 3 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2022/competition/)!
+
+Before then, we have a few important updates and reminders for you:
+
+## COVID Safety
+
+Life is beginning to get back to normal, and this means we are finally able to do in person events again! However, we must recognise that some people still have to be cautious, to protect their own health or their friends or family. As such, we ask that everyone follows [our COVID policy](https://studentrobotics.org/covid-19/) to help ensure we all keep safe, whilst still having lots of fun with robots!
+
+Please keep an eye on this as we get closer to the competition, as the content may change.
+
+## Robot design considerations
+
+At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:
+
+- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.
+- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this.
+- For safety, please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.
+- Ensure your battery is protected, ideally housed in its own rigid container
+- Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.
+
+All of these and more can be found in our [safety regulations](https://studentrobotics.org/docs/kit/safety-regulations) and [rules](https://studentrobotics.org/rules).
+
+## Southampton Marathon
+
+On Sunday 24th April, Southampton is holding a marathon event, the route of which goes right through the university.
+
+When arriving on Sunday morning, please be prompt and arrive as close as possible to the doors opening. This not only ensures there will be parking, but that the campus won't be crowded with the general public. Additionally, it's possible you will only be able to approach the university from the west, due to the route closing part of University. The schedule is available on [our website](https://studentrobotics.org/events/sr2022/competition/).
```

Comment:
This whole marathon section feels inaccurate, especially as the details are still being handled.

We can't guarantee what the state of the parking will be or how much public will be around, or how many marathon organisers. To be honest, I think it could be quite busy regardless.
It is guaranteed they will only be able to approach from the west.
Nit pick: "Southampton" is not holding a marathon event, though I grant you it is hosting one.

I would suggest just saying:
"""
On Sunday 24th April a marathon event is being held in Southampton. As a result there will be a number of road closures in the area, including on the university campus. Parking will also be limited. Keep an eye on the competition page on our website to see up-to-date information on how to reach the competition.

Access to food sellers may also be restricted, so please plan accordingly.
"""